__tests__/posts.ts

```diff
@@ -2169,6 +2169,156 @@ describe('mutation checkLinkPreview', () => {
   });
 });
 
+describe('mutation createPost', () => {
```

Comment:
Let's add a test to check that we don't allow creating a post in a squad I don't belong to

Reply:
And another test for mentioning people outside the squad. This should not be allowed